addon/mixins/adapter-api-host-proxy.js

```diff
@@ -0,0 +1,23 @@
+/**
+  @module ember-jsonapi-resources
+  @submodule adapter-api-host-proxy-mixin
+**/
+import Ember from 'ember';
+
+/**
+  Mixin to provide url rewrite for proxied api. Mostly used as example.
+
+  @class AdapterApiHostProxyMixin
+  @static
+*/
+export default Ember.Mixin.create({
```

Comment:
My pleasure. When I find the time I'll look through all the docs. Those needs some touching up I reckon.